File: SecuJournal/sjDate.cs
using System;
//using System.Collections.Generic;
//using System.ComponentModel;
//using System.Data;
//using System.Drawing;
//using System.Linq;
//using System.Text;
//using System.Threading.Tasks;
using System.Windows.Forms;

namespace SecuJournal
{
    public partial class sjDate : Form
    {
        public sjDate()
        {
            InitializeComponent();
        }

        private void UCDTButton_Click(object sender, EventArgs e)
        {
            Clipboard.SetText(DateTime.Now.ToString());
            sjEntry sje = new sjEntry();
            sje.RichTextBox1.Paste();
            this.Close();
        }

        private void CancelButton_Click(object sender, EventArgs e)
        {
            Clipboard.SetText(DateTimePicker1.Text + " " + DateTimePicker2.Text);
            sjEntry sje = new sjEntry();
            sje.RichTextBox1.Paste();
            this.Close();
        }

        private void OKButton_Click(object sender, EventArgs e)
        {
            this.Close();
        }
    }
}

File: SecuJournal/SJDate.vb
Public Class SJDate
    Private Sub UCDTButton_Click(sender As Object, e As EventArgs) Handles UCDTButton.Click
        Clipboard.SetText(Date.Now.ToString)
        SJEntry.RichTextBox1.Paste()
        Me.Close()
    End Sub

    Private Sub OKButton_Click(sender As Object, e As EventArgs) Handles OKButton.Click
        Clipboard.SetText(DateTimePicker1.Text + " " + DateTimePicker2.Text)
        SJEntry.RichTextBox1.Paste()
    End Sub

    Private Sub CancelButton_Click(sender As Object, e As EventArgs) Handles CancelButton.Click
        Me.Close()
    End Sub
End Class